src/clj/game/core/actions.clj

```diff
@@ -402,7 +402,8 @@
                 (dotimes [n times-pump]
                   (resolve-ability state side (dissoc pump-ability :cost :msg) (get-card state card) nil))
                 (doseq [sub (remove :broken (:subroutines current-ice))]
-                  (break-subroutine! state (get-card state current-ice) sub))
+                  (break-subroutine! state (get-card state current-ice) sub)
+                  (continue-ability state side (:additional-ability break-ability) (get-card state card) nil))
```

Comment:
This can't use `continue-ability` cuz it'll complete the eid, and we don't want to complete it until later.